NCS.DSS.AzureSearchUtility/RunIndex.cs

```diff
@@ -1,19 +1,21 @@
-﻿using NCS.DSS.AzureSearchUtilities.CreateIndex;
-using NCS.DSS.AzureSearchUtilities.Models;
-using Newtonsoft.Json;
-using System;
+﻿using System;
 using System.IO;
+using NCS.DSS.AzureSearchUtility.CreateIndex;
+using NCS.DSS.AzureSearchUtility.Helpers;
+using NCS.DSS.AzureSearchUtility.Models;
+using Newtonsoft.Json;
 
-namespace NCS.DSS.AzureSearchUtilities
+namespace NCS.DSS.AzureSearchUtility
 {
     public class RunIndex
     {
-        /// <param name="args"> Command line arguments: /SearchAdminKey:blah /CosmosConnString:"AccounEndpoint=https://proj-env-shared-cdb.documents.azure.com:443/;AccountKey=secretblah;Database=customers;"  /SearchConfigFile:pathtoblah</param>
+        /// <param name="args"> Command line arguments: /SearchAdminKey:blah /SearchConfigFile:pathtoblah /EnvironmentName:AT</param>
         public static void Main(string[] args)
         {
             var searchAdminKey = string.Empty;
-            string cosmosConnectionString = string.Empty;
-            string searchConfigFile = string.Empty;
+            var searchConfigFile = string.Empty;
+            var environmentName = string.Empty;
+            var synonymPath = string.Empty;
 
             if (args.Length == 0)
                 throw (new NotSupportedException("Missing arguments"));
```

Comment:
Is there some way to simplify this conditional? Maybe take a look at https://www.nuget.org/packages/Microsoft.Extensions.CommandLineUtils